simulator/src/main/cpp/ChirpFMRxTx.h

```diff
@@ -2,7 +2,14 @@
 class ChirpFMRxTx
 {
     public:
-        virtual void send(int, char[], size_t) = 0;
-        virtual char* receive() = 0;
-        virtual int getToAddress(char[]) = 0;
+        virtual void send(int, char[], size_t);
```

Comment:
Same question as for the bluetooth interface, is this class no longer just an interface?

Reply:
same answer as above. reverted.